ipapython/ipaldap.py

```diff
@@ -1068,7 +1069,8 @@ def _connect(self):
 
         return conn
 
-    def simple_bind(self, bind_dn, bind_password, server_controls=None,
+    def simple_bind(self, bind_dn=DN(('cn', 'directory manager')),
+                    bind_password='', server_controls=None,
```

Comment:
I don't think I've encountered a place in the code where `bind_dn` would be anything other than `cn=directory manager`. I think having sane defaults is a good approach in that case.